resources/assets/components/CampaignsTable.js

```diff
@@ -81,11 +47,11 @@ const CampaignsTable = ({ isOpen, filter }) => {
   const [orderBy, setOrderBy] = useState('pending_count,desc');
 
   const { error, loading, data, fetchMore } = useQuery(CAMPAIGNS_QUERY, {
-    variables: { isOpen, orderBy },
+    variables: { filter, isOpen /* orderBy */ },
```

Comment:
One problem I ran into is that Algolia (somewhat surprisingly) [does not support sorting](https://www.algolia.com/doc/guides/managing-results/refine-results/sorting/#backend-implementation-replicas) results out of the box. The suggested workaround seems to be to create separate replica indexes for each possible sort method (e.g. one where items are sorted by ID, one where they're sorted by title, etc.)

This is probably not a deal-breaker, but is worth considering when estimating our usage & definitely worth at least figuring out a proof-of-concept of here so we know how feasible this is for future product builds.

Reply:
Another option – if we don't plan to use sorting elsewhere – would be to separate _paging_ through campaigns (via the existing `paginatedCampaigns` query) from _searching_ for campaigns (which could maybe display in a separate "search auto-fill" interface when using the search field, and not be sortable).